code/game/objects/items/storage/pouch.dm

```diff
@@ -10,12 +10,15 @@
 	allow_drawing_method = TRUE
 	var/fill_type
 	var/fill_number = 0
+	///Defines how many versions of the empty sprite there are
+	var/sprite_slots = null
 
 /obj/item/storage/pouch/Initialize(mapload)
 	. = ..()
 	if(fill_number && fill_type)
 		for(var/i in 1 to fill_number)
 			new fill_type(src)
+	update_icon_state()
```

Comment:
you should never be calling update_icon_state directly, just update_icon

Reply:
Gotcha. It's update_icon() now.